include/fiction/algorithms/simulation/sidb/sidb_simulation_result.hpp

```diff
@@ -59,6 +63,59 @@ struct sidb_simulation_result
      * The key of the map is the name of the parameter, the element is the value of the parameter.
      */
     std::unordered_map<std::string, std::any> additional_simulation_parameters{};
+    /**
+     * This function calculates the ground state charge distributions from the provided simulation results.
+     * The ground state charge distributions are those with energy closest to the minimum energy found in the simulation
+     * results.
```

Comment:
What do we mean by "energy closest to the minimum found in the simulation results"? Don't we return **the** minimum ones?

Reply:
good point! It is misleading here. I will change it! Thanks.